clusterloader2/pkg/measurement/util/resource.go

```diff
@@ -0,0 +1,127 @@
+/*
```

Comment:
Would it be possible to split this file to a separate PR - it seems to be independent and for me review time is exponential from the PR size :)

Reply:
File moved to #240